App.tsx

```diff
@@ -45,21 +45,39 @@ const App: FunctionComponent = () => {
   }, [])
 
   const initializeMatomo = () => {
-    const displayAnalytics = env.DISPLAY_ANALYTICS
     const matomoUrl = env.MATOMO_URL
     const matomoSiteId = parseInt(env.MATOMO_SITE_ID)
-    if (displayAnalytics) {
+
+    if (matomoUrl && !isNaN(matomoSiteId)) {
```

Comment:
Is it worth giving `!isNaN(matomoSiteId)` a name?

Reply:
i think so!